### ISSUE TITLE
A legit furnace cart?

### ISSUE THREAD

**Issue by Prospector** [BrassGoggledCoders/MoarCarts#10]
So in vanilla there's a furnace cart, but it's not used to smelt, it's used to speed up carts. I'm not sure how, but it would be nice if there was an actual furnace cart that was used to cook items.

**Comment by hron84** [comment]
Only one thing: how do you want to handle fuels that are also cookable, like coal with BigReactor or saplings with Witchery installed?

**Comment by SkySom** [comment]
It'd be it's own thing. Not a part of the current furnace cart.

**Comment by hron84** [comment]
@SkySom It's clear for me, I popped up the question in general meaning. Or you can just simply pass the entire behavior to the actual furnace inventory handler code?

**Comment by SkySom** [comment]
It'd be like using the actual furnace

**Comment by hron84** [comment]
It could be a bit more tricky. Actual furnace is strict on sides, and I think you cannot insert thing from the back of the minecart (furnace accepts the fuel only from its back), because... you know... define the "back" of the minecart. Also, it could mess up if you want to integrate with Railcraft item loaders what has no sneaky inserting feature or slot selector.

**Comment by SkySom** [comment]
Not really. Yes inserting may be weird (As all MoarCarts try to insert from the top and pull from the bottom), and the back of the minecart is the back of the minecart, nothing hard to define there. and Railcraft item loaders just try to ensure into a normal inventory. It's up to a cart to decide how to insert things.
